examples/map_transformations/projection_custom_origin.py

```diff
@@ -0,0 +1,87 @@
+"""
+=====================================================
+Reprojecting to a Map Projection with a Custom Origin
+=====================================================
+
+In this example, we show how to reproject a map to a map projection with a
+custom origin.  Here, we choose the target map projection to be the
+`azimuthal equidistant projection <https://en.wikipedia.org/wiki/Azimuthal_equidistant_projection>`__,
+also known as the Postel projection, which has useful properties relative to a
+specified origin of the projection.  If a different map projection is desired,
+modifying this example is straightforward.
+
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+###############################################################################
+# We will use one of the AIA images from the sample data.  We fix the range of
+# values for the Map's normalizer for a prettier image.
+
+aia_map = sunpy.map.Map(AIA_171_IMAGE)
+aia_map.plot_settings['norm'].vmin = 0
+aia_map.plot_settings['norm'].vmax = 10000
+
+###############################################################################
+# Next, we create a `~astropy.coordinates.SkyCoord` to define the custom origin
+# of the map projection.  Here, we are going to center the projection at the
+# helioprojective coordinates of a particular active region.  We want our map
+# projection to be in heliographic Stonyhurst coordinates, so we transform the
+# origin coordinate accordingly.
+
+origin_hpc = SkyCoord(735*u.arcsec, -340*u.arcsec, frame=aia_map.coordinate_frame)
+origin = origin_hpc.heliographic_stonyhurst
+
+###############################################################################
+# We then create a FITS-WCS header that includes our custom origin coordinate.
+# The azimuthal equidistant projection is specified by the code `"ARC"`.
+# See :ref:`astropy:supported_projections` for the projection codes for other
+# projections.
```

Comment:
```suggestion
# See :doc:`astropy:/wcs/supported_projections` for the projection codes for
# other projections.
```
Sorry, I gave you a broken intersphinx link.  I think this fix will work, but I can't test locally right now.